### PR TITLE
Like system for forum posts

### PR DESCRIPTION
**Pull request by rgilman**
This addresses #42 .

### CODE REVIEW

File: assets/js/like.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
+jQuery(document).ready( function() {
+
+   jQuery(".bfc-post-like").click( function(event) {
+		var this2 = this
```
Comment (iangilman) [inline assets/js/like.js:4]: I don't know if you want to be bothered with such things, but there are various places in this file where the indents aren't right or semicolons are missing. Neither cause functional problems, but it's still good hygiene to keep everything tidy.

**Review by iangilman — COMMENTED** [review]
Exciting! This is a big one 😍

File: assets/js/like.js (hunk 1 above)
Comment (rgilman) [reply assets/js/like.js:4]: Thanks for catching the semicolons. I was only able to find one indent issue, with` jQuery(".bfc-post-like").click( function(event) {`

I agree. Good to keep it tidy.

Comment (iangilman) [reply assets/js/like.js:4]: Cool. Here's what I'm seeing on the indent front (I'm assuming you are going for two space indents):

Line 3 has three spaces where it should have just two.

Lines 5 through 7 have six spaces when they should have four.

Looks like there are some tabs mixed in with the spaces… it's generally best practice to stick with one or the other. I personally prefer spaces. Depending on what editor you're using, you should be able to get it to do it automatically for you. 

The rest of it looks good :-)

Comment (rgilman) [reply assets/js/like.js:4]: Thanks for this next level of catching these. Ah, the perils of copying code from elsewhere. We do have an editorconfig file which is set to indent_style = tabs and indent_size = 4. I've cleaned up my local file to that standard and will upload it in due course.

Comment (iangilman) [reply assets/js/like.js:4]: Sounds good!

**Pull request merged**